mods/cnc/audio/notifications.yaml

```diff
@@ -4,49 +4,76 @@ Speech:
 		nod: nod
 	Notifications:
 		AirstrikeReady: airredy1
+			Transcription: Airstrike ready.
 		BaseAttack: baseatk1
+			Transcription: Our base is under attack.
 		Building: bldging1
 		BuildingCannotPlaceAudio: deploy1
+			Transcription: Cannot deploy here.
 		BuildingCaptured: capt1
+			Transcription: Building captured.
 		BuildingInProgress: bldg1
+			Transcription: Unable to comply. Building in progress.
 		BuildingLost: strclost
+			Transcription: Structure lost.
 		Cancelled: cancel1
 		CivilianBuildingCaptured: civcapt1
+			Transcription: Civilian building captured.
 		CivilianKilled: civdead1
+			Transcription: Civilian killed.
 		ConstructionComplete: constru1
+			Transcription: Construction complete.
 		EnemyUnitsApproaching: enmyunit
+			Transcription: Enemy approaching.
```

Comment:
`enmyunit` actually says "Enemy unit destroyed." Seems like you transcribed `enmyapp1`?

Reply:
The announcement name is "EnemyUnitsApproaching" so this is just highlighting an earlier mistake.

Reply:
`EnemyUnitsApproaching` is not used anywhere and probably that's why it was not spotted until now. Should I take any action here?